examples/demo/src/orders/OrderEdit.tsx

```diff
@@ -84,143 +84,152 @@ const OrderForm = (props: any) => {
     return (
         <FormWithRedirect
             {...props}
-            render={(formProps: any) => (
-                <Box maxWidth="50em">
-                    <Card>
-                        <CardContent>
-                            <Grid container spacing={1}>
-                                <Grid item xs={12} sm={12} md={8}>
-                                    <Typography variant="h6" gutterBottom>
-                                        {translate(
-                                            'resources.commands.section.order'
-                                        )}
-                                    </Typography>
-                                    <Grid container>
-                                        <Grid item xs={12} sm={12} md={6}>
-                                            <Labeled
-                                                source="date"
-                                                resource="commands"
-                                            >
-                                                <DateField
+            render={({ handleSubmit, ...formProps }: any) => (
+                <form onSubmit={handleSubmit}>
```

Comment:
That's a breaking change, too, and a big one. I thought SimpleForm and others included a `<form>` tag. Can you revert this BC break?

Reply:
`SimpleForm` and `TabbedForm` includes the `form` tag but here, we're using `FormWithRedirect`. No we can't remove this BC break as it is required to ensure native submit on enter for instance